packages/host/app/services/ai-assistant-panel-service.ts

```diff
@@ -226,6 +289,22 @@ export default class AiAssistantPanelService extends Service {
         let { roomId } = await createRoomCommand.execute(input);
 
         window.localStorage.setItem(NewSessionIdPersistenceKey, roomId);
+
+        // If file history should be copied, send an initial message with the files and cards
+        if (shouldCopyFileHistory) {
+          const { attachedFiles, attachedCards } =
+            this.collectFileHistoryFromCurrentRoom();
+
+          if (attachedFiles.length > 0 || attachedCards.length > 0) {
+            await this.matrixService.sendMessage(
+              roomId,
+              'This session includes files and cards from the previous conversation for context.',
+              attachedCards,
+              attachedFiles,
+            );
+          }
```

Comment:
Is there anything to stop this message from generating a response from the aibot?